libraries/SITL/SIM_Gripper_EPM.cpp

```diff
@@ -94,6 +115,6 @@ float Gripper_EPM::tesla()
 {
     // https://en.wikipedia.org/wiki/Orders_of_magnitude_(magnetic_field)
     // 200N lifting capacity ~= 2.5T
-    const float percentage_to_tesla = 0.25;
-    return percentage_to_tesla * field_strength/100.0f;
+    const float percentage_to_tesla = 0.25f;
+    return static_cast<float>(percentage_to_tesla * field_strength/100.0f);
```

Comment:
Some people aren't really keen on these static-casts.  Adding them into SITL-only code probably isn't the best of ideas.